src/test/java/org/fogbowcloud/manager/OrderStateTransitionerTest.java

```diff
@@ -60,21 +58,32 @@ public void tearDown() {
         }
     }
 
+    // test case: When calling the transition() method passing an Order and a OrderState, this
+    // order will be change to the specific OrderState, removing of the origin list, and adding in
+    // that state list.
     @Test
     public void testValidTransition() throws UnexpectedException {
```

Comment:
note that we test only the open to spawning transition. I suggest to change the name of the method to better reflect what it does.

Reply:
... change the description too.

Reply:
Done.